ngraph/test/type_prop/experimental_detectron_prior_grid_generator.cpp

```diff
@@ -176,3 +176,99 @@ TEST(type_prop, detectron_grid_generator_dynamic_shapes_intervals)
         ASSERT_TRUE(grid_gen->get_output_partial_shape(0).same_scheme(s.ref_out_shape));
     }
 }
+
+TEST(type_prop, detectron_grid_generator_dynamic_shapes_intervals_1)
```

Comment:
Please use parametrized test.

Reply:
Fixed.

Reply:
As I see you still don't have parametrized test.

Reply:
Done.